Eatery/Controllers/Eateries/Campus/ScrollableViewController.swift

```diff
@@ -0,0 +1,283 @@
+//
+//  ScrollableViewController.swift
+//  Eatery
+//
+//  Created by Sergio Diaz on 12/9/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class ScrollableViewController: UIViewController {
+
+    private let eatery: CampusEatery!
+    private let viewControllers: [CampusEateryExpandedMenuViewController]!  // The VCs for each menu section
+    private let originalViews: [UIView]!    // The views of the VCs for each menu section
+    private let allItems: [ExpandedMenu.Item]!  // All menu items combined into array and sorted
+
+    private var headerContainer: ExpandedHeaderContainer!
+    private var tabBar: TabBar!
+    private var tabBarStack: UIStackView!
+    private var menuItemsView: ExpandedMenuItemView!
+
+    var scrollView: UIScrollView?
+    var scrollOffset: CGFloat = 0
+    private var fillerSpaceHeight: CGFloat = 0
+
+    private var beingMoved = false
+    private var isManualScrolling = false
+    private var orderedViewsCreated = false
+
+    init(eatery: CampusEatery, viewControllers: [CampusEateryExpandedMenuViewController], items: [ExpandedMenu.Item]) {
+        self.eatery = eatery
+        self.viewControllers = viewControllers
+        self.originalViews = viewControllers.map { $0.view }
+        self.allItems = items.sorted { item1, item2 in
+            if item1.getNumericPrice() > item2.getNumericPrice() {
+                return false
+            }
+            return true
```

Comment:
I think this can be simplified to `item1.getNumericPrice() <= item2.getNumericPrice()`? From there the [docs](https://developer.apple.com/documentation/swift/array/2296815-sorted) for `sorted(by:)` mention that the closure should be irreflexive, so the final should be  `item1.getNumericPrice() < item2.getNumericPrice()`